katas/content/phase_estimation/eigenvectors_x/Verification.qs

```diff
@@ -0,0 +1,32 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Math;
+
+    @EntryPoint()
+    operation CheckSolution() : Bool {
+        let actual = Kata.EigenvectorsX();
+        if Length(actual) != 2 {
+            Message("The array of eigenvectors should have exactly two elements.");
+            return false;
+        }
+        for i in 0 .. 1 {
+            if Length(actual[i]) != 2 {
+                Message("Each eigenvector should have exactly two elements.");
+                return false;
+            }
+            if AbsD(actual[i][0]) + AbsD(actual[i][1]) < 1E-9 {
```

Comment:
There's 0.001 tolerance a bit earlier. Should those be unified or there's a reason to keep them different?

Reply:
In both tasks the return values are hardcoded rather than computed, so the check tolerance is driven by gut feeling rather than by exact science. 

The 0.001 tolerance was for the eigenvalues task (it's the same one used in complex arithmetic kata); there, the answer is specific values, and we're checking to see that those values were entered (the other option is completely incorrect values, rather than values that are slightly off; the learner is extremely unlikely to enter something like 1.005). 

The 1E-9 tolerance is for the task that involves any non-zero vectors; vectors like [0.0001, 0.0001] are valid eigenvectors, so with the check I want to make sure that the learner actually entered a vector that satisfies the condition and didn't leave [0, 0] in there (that vector also happens to satisfy the condition, but it's not an eigenvector because it's zero).